### PR TITLE
fix: changing view in create multisig vault

### PR DESCRIPTION
**Pull request by konradodo**

### CODE REVIEW

File: src/popup/components/MultisigVaultCreateReview.vue
Comment (peronczyk) [inline src/popup/components/MultisigVaultCreateReview.vue:1]: Remove the `review-details-row` from the CSS block.

File: src/popup/components/MultisigVaultCreateReview.vue [hunk 1]
```diff
@@ -22,37 +22,38 @@
...
+      <template #value>
+        <div class="authorized-signers">
+          <div
+            v-for="signer in signers"
+            :key="signer.address"
+            class="authorized-signers-row"
+          >
+            <AccountItem
+              :address="signer.address"
+              :protocol="PROTOCOL_AETERNITY"
```
Comment (peronczyk) [inline src/popup/components/MultisigVaultCreateReview.vue:46]: As far as I understood the protocol icon should be removed from signers.

**Review by peronczyk — APPROVED** [review]

**Review by CedrikNikita — APPROVED** [review]

**Comment by Liubov-crypto** [comment]
LGTM
в

**Comment by CharisSiarampalis** [comment]
Lgtm

**Pull request merged**